sig-docs/image-management/image-management.md

```diff
@@ -0,0 +1,37 @@
+# Image Management
+
+You can insert images in your files where they can be accessed anywhere, creating a consistent experience for your readers. Here is an example of how you can use PicGo to pull images from QingStor.
+
+## Getting Started
+
+- You can check the [official introduction](https://picgo.github.io/PicGo-Doc/en/) of PicGo or click [here](https://github.com/Molunerfinn/PicGo/releases) to install it.
+- You need an account to use object storage services (the example here is QingStor).
+
+## Step
+
+- Log in the QingCloud [console](https://console.qingcloud.com/login) and click Access Key in the drop-down menu at the top right corner.
```

Comment:
It would be better to use the serial number for each sentence instead of the bullet list.